### DISCUSSION TITLE
Welcome to shopify-packer Discussions!

### DISCUSSION THREAD

**Discussion by hayes0724** [hayes0724/shopify-packer#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MykalMachon** [comment]
Hey! I'm Mykal Machon, I work on Shopify quite a bit for freelance work.

**Just wanted to hop in and say that Shopify-Packer looks great!** 
I've tried making my own, admittedly much simpler, versions of a Shopify development tool since slate was deprecated. _My current solution works_, but I always felt like I was missing this or that.
 
Shopify-Packer seems to address everything that I'd want my own solutions to do- nice work! 

I'm looking forward to trying out Shopify-Packer in a project sometime soon!
